Specs/Scene/ImplicitTileCoordinatesSpec.js

```diff
@@ -4,9 +4,30 @@ import {
 } from "../../Source/Cesium.js";
 
 describe("Scene/ImplicitTileCoordinates", function () {
+  function internalQuadtree(level, x, y) {
```

Comment:
why the name `internal`? I thought you meant internal node, but I've seen you use it with (0, 0, 0).

Reply:
I was mimicking after `internalQuadtreeDescription` in `ImplicitSubtreeSpec`. I used to have it be `buildQuadtree` so maybe I'll set it back to that to avoid confusion.

Reply:
ohhh that "internal" meant subtree with internal buffers (rather than external buffers referenced by uri). not relevant here.

maybe `quadtreeCoordinates`? `buildQuadree` sounds like you're making a whole tree of nodes